config/locales/blacklight.en.yml

```diff
@@ -16,6 +16,9 @@ en:
         title: 'Refine search results'
       continue_search:
         title: 'Or continue your search in'
+      zero_results:
+        remove_fields: ''
```

Comment:
@ruthtillman is `remove_fields` supposed to be empty, if so why are we adding it?

Reply:
Sorry, y'all, I'd saved but AFTER the commit for some reason. Pulled, committed, updated, pushed... hoping the newest version is now ok.